corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/binding/Bindings.kt

```diff
@@ -1,10 +1,47 @@
 package io.github.droidkaigi.confsched2020.binding
 
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
 import android.view.View
 import androidx.core.view.isVisible
 import androidx.databinding.BindingAdapter
 
 @BindingAdapter("isVisible")
-fun View.showHide(show: Boolean) {
+fun View.showGone(show: Boolean) {
     isVisible = show
 }
+
+@BindingAdapter("isHide")
+fun View.showHide(show: Boolean) {
+    visibility = if (show) View.VISIBLE else View.INVISIBLE
+}
+
+@BindingAdapter("isVisibleWithAnimation")
+fun View.showGoneWithAnimation(show: Boolean) {
+    val shortAnimationDuration = resources.getInteger(android.R.integer.config_longAnimTime)
```

Comment:
[nit picking] the variable name says `short` but the resource name says `long` so could you rename the variable or use the shorter resource (I think either is fine)?